devenv.md

```diff
@@ -0,0 +1,12 @@
+# devenv
```

Comment:
What do you think about to move this file to another MR? One for the devenv documentation.